lib/passport-configurator.js

```diff
@@ -300,16 +300,18 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
                 }
               } else {
                 user.hasPassword(password, function(err, ok) {
```

Comment:
should probably set `var errorMsg = 'Invalid username and/or password and/or email has not been Verified.'` here and user the variable instead of duplicating down below.